data/mods/MindOverMatter/mutations/psi_passives.json

```diff
@@ -32,9 +32,15 @@
     "enchantments": [
       {
         "values": [
-          { "value": "HUNGER", "multiply": { "math": [ "(-0.03 - (0.015 * u_school_level('BIOKINETIC')))" ] } },
-          { "value": "THIRST", "multiply": { "math": [ "(-0.03 - (0.015 * u_school_level('BIOKINETIC')))" ] } },
-          { "value": "METABOLISM", "multiply": { "math": [ "(-0.03 - (0.015 * u_school_level('BIOKINETIC')))" ] } }
+          { "value": "HUNGER", "multiply": { "math": [ "(-0.03 - (0.00075 * u_spell_level_sum('school': 'BIOKINETIC')))" ] } },
```

Comment:
Few strings above there is vitamin_absorb_multi, and i want to point that there is VITAMIN_ABSORB_MOD enchantment, that behave the same

Reply:
Oh right, good chance to replace that.